docs/asb/README.md

```diff
@@ -0,0 +1,134 @@
+# XMR to BTC Atomic Swap - Automated Swap Backend (ASB)
+
+## Quick Start ASB
+
+- Ensure that you have the Monero Wallet RPC running with `wallet-dir` and `disable-rpc-login`:
```

Comment:
I'd use the literal options here to make it obvious that we are talking about commandline options.

```suggestion
- Ensure that you have the Monero Wallet RPC running with `--wallet-dir` and `--disable-rpc-login`:
```